catalyst/utils/metrics/mrr.py

```diff
@@ -0,0 +1,46 @@
+"""
+MRR metric.
+"""
+
+import torch
+
+
+def mrr(outputs: torch.Tensor, targets: torch.Tensor, k=100) -> torch.Tensor:
+
+    """
+    Calculate the MRR score given model ouptputs and targets
```

Comment:
I'd appreciate it if you could extend documentation with the explanation of the metric or add link users will be able to read more about this score.